### PR TITLE
Exercise 10

### PR DESCRIPTION
**Pull request by aleyipsoftwire**

### CODE REVIEW

File: ansible/vars/env.yml
Comment (aleyipsoftwire) [inline ansible/vars/env.yml:1]: I've intentionally deleted this without regenerating the values, since we're not working with ansible anymore

Comment (Jonesey13) [reply ansible/vars/env.yml:1]: Fair enough!

File: README.md
Comment (Jonesey13) [inline README.md:1]: There are still a couple of references to Trello in this file

File: todo_app/data/mongo_items.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import os
+from enum import Enum
+
+import pymongo
+from bson import ObjectId
+
+
+class ItemStatus(Enum):
+    TO_DO = 'TO_DO'
+    DONE = 'DONE'
```
Comment (Jonesey13) [inline todo_app/data/mongo_items.py:10]: If you were using Python 3.11 or greater `StrEnum` may have been useful here (also could arguably be moved to its own file).

**Review by Jonesey13 — APPROVED** [review]
Looks good to me! Comments below are minor (feel free to merge in when you are ready).

**Pull request merged**